src/redux/store.js

```diff
@@ -0,0 +1,11 @@
+import { configureStore } from '@reduxjs/toolkit';
+import bookReducer from './books/booksSlice';
+// import categoriesReducer from './categories/categoriesSlice';
```

Comment:
- [ ] Kindly consider removing this line of commented code as it makes our codebase looks ugly. removing it will make your codebase professional.